### ISSUE TITLE
Improve the contrast of the buttons: WCAG

### ISSUE THREAD

**Issue by MxTheo** [MxTheo/travelingguestbook#46]
The app should become more accessible. Color contrast is low. White text on dark background

References:
https://www.google.com/accessibility/for-developers/
https://www.w3.org/WAI/standards-guidelines/wcag/
https://m3.material.io/

Contrast checker
https://webaim.org/resources/contrastchecker/
https://developer.paciellogroup.com/color-contrast-checker/
https://www.color-blindness.com/coblis-color-blindness-simulator/
